docker-compose-allure.yml

```diff
@@ -1,11 +1,18 @@
 ---
 version: '3.7'
 
+networks:
+  default:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.16.57.0/24
```

Comment:
otherwise docker failing in case you have active VPN connection

Reply:
this should be also done on docker-compose.yml?